### ISSUE TITLE
Convert project files to conventional format

### ISSUE THREAD

**Issue by oSumAtrIX** [ReVanced/revanced-branding#4]
## Issue

Currently the project files are saved in an unconventional format `.afdesign` which can only be opened ny Affinity Designer. 

## Solution

Convert all project files to a conventional format such as `.psd` or `.ai`.

## Motivation

A minority owns Affinity Designer, but Affinity Designer can open other formats such as `.psd` and `.ai`.

**Comment by Ushie** [comment]
Why not SVG?

**Comment by oSumAtrIX** [comment]
#3 for example demonstrates why it is not a good idea to use rendered files as source files. See it this way, you compile your code to a binary. While it might be readable and technically usable when decompiled, you can not infer the full source of the binary easily. The same way #3 demonstrates, how the "compiled" SVG file might lose aspects of it's original source.

**Comment by oSumAtrIX** [comment]
Seems like exporting to `.ai` is not an option for Affinity Designer. Alternatively, this exists:
![Export to `.psd`](https://user-images.githubusercontent.com/13122796/219533656-b183c1ba-716d-40de-a4d8-ddd5a742ad7c.png)

From there the project file can be converted to `.ai`.

**Comment by kazimmt** [comment]
> Seems like exporting to `.ai` is not an option for Affinity Designer.

Just convert to `.eps`

**Comment by oSumAtrIX** [comment]
Is eps "lossless" in terms of editability? Can project files be converted to eps and back to project files without any loss?

**Comment by kazimmt** [comment]
> Is eps "lossless" in terms of editability? Can project files be converted to eps and back to project files without any loss?

YES

**Comment by kazimmt** [comment]
But gradients will convert to image
That's why i made it manually in my last PR

**Comment by oSumAtrIX** [comment]
Then it is not lossless.

**Comment by ArjixWasTaken** [comment]
> #3 for example demonstrates why it is not a good idea to use rendered files as source files. See it this way, you compile your code to a binary. While it might be readable and technically usable when decompiled, you can not infer the full source of the binary easily. The same way #3 demonstrates, how the "compiled" SVG file might lose aspects of it's original source.

What if you just treat the SVGs as the source files, and use a program to convert all text to shapes/paths?
That way you have SVGs with text, but distribute SVGs with compiled text.

**Comment by oSumAtrIX** [comment]
SVG files are not source files. They are rendered assets. They can be lossy in terms of of the source that produced them.

**Comment by ArjixWasTaken** [comment]
Yes but they are not a raster format, so the loss is not in quality.

**Comment by oSumAtrIX** [comment]
The loss is in information, not quality.

**Comment by oSumAtrIX** [comment]
It turns out that there is no practical way to convert the project files to conventional source formats such as `.ai` or `.psd`. Technically speaking, a specification of the logos can be published, which can be used to recreate the source files in any format.

**Comment by ArjixWasTaken** [comment]
> The loss is in information, not quality.

One could debate that the loss only happens when using an SVG editor.

Handwritten SVGs don't have that issue.

But I do understand your point.

**Comment by Ushie** [comment]
What's the issue in using SVGs as source files? there are programs like Inkscape which specialise in this and they are as dependable as any other editor or format

**Comment by oSumAtrIX** [comment]
The problem isn't about SVGs particularly; the information about the original source is lost when rendered as SVG. As an example, the source font is lost.

**Comment by oSumAtrIX** [comment]
Besides, the SVG files are treated as rendered assets like compiled code. Your argument is similar to saying that you can still read compiled code in its fullest since you can disassemble it, but the source code is lost.

**Comment by ArjixWasTaken** [comment]
> The problem isn't about SVGs particularly; the information about the original source is lost when rendered as SVG. As an example, the source font as a form of information delivered by the source is lost when rendering the SVG as paths. 

SVGs are just plain XML files, when handwritten you can place comments like that.

**Comment by oSumAtrIX** [comment]
I am aware, as I explained, that the issue isn't the SVG files particularly. You can also simply write your compiled programs bit by bit instead of compiling their source.

**Comment by ArjixWasTaken** [comment]
> I am aware, as I explained, that the issue isn't the SVG files particularly. You can also simply write your compiled programs bit by bit instead of compiling their source. 

Ah yeah, asm supports comments

This is golden

**Comment by oSumAtrIX** [comment]
That being said, the source should be delivered in a traditional and sane format such as explained in the original issue.
